Library/test-helpers/lib.sh

```diff
@@ -2142,8 +2074,7 @@ limeconRunRegistrar() {
     local IP=$3
     local NETWORK=$4
 
-
-    podman run -d --name $NAME --net $NETWORK --ip $IP --cap-add CAP_AUDIT_WRITE --cap-add CAP_SYS_CHROOT --volume=/etc/keylime/:/etc/keylime/ --volume=$PWD/cv_ca:/var/lib/keylime/cv_ca:z localhost/$TAG /usr/local/bin/lime_con_start /usr/bin/keylime_registrar
+    limeconRun $NAME $TAG $IP $NETWORK "/usr/local/bin/lime_con_start /usr/bin/keylime_registrar" "--volume=/etc/keylime/:/etc/keylime/ --volume=$PWD/cv_ca:/var/lib/keylime/cv_ca:z"
```

Comment:
I just realized, sometimes keylime scripts can be in `/usr/local/bin`, we should probably address that somehow.
We could e.g. pass only the script name (`keylime_registrar`) and do the path detection in the `lime_con_start` script (a.k.a. `SCRIPT_PATH=$( which $SCRIPT )`). Maybe it is not necessary to fix it right away but you will probably hit this problem when working on the kontainer with upstream keylime.

Reply:
I will deal with that in a separate commit later.